pkg/render/egressgateway/egressgateway.go

```diff
@@ -153,6 +153,10 @@ func (c *component) deploymentPodTemplate() *corev1.PodTemplateSpec {
 	for _, x := range c.config.PullSecrets {
 		ps = append(ps, corev1.LocalObjectReference{Name: x.Name})
 	}
+	tolerations := []corev1.Toleration{}
```

Comment:
This will leave a warning in goland IDE, best to replace with:
```
var tolerations []corev1.Toleration
```

Reply:
Fixed.